DOCS-435 update Monitoring.md (#98)
aerospike/kubernetes-operator: docs/Monitoring.md

  podSpec:
    sidecars:
     - name: aerospike-prometheus-exporter
       image: aerospike/aerospike-prometheus-exporter:1.3.0
       ports:
         - containerPort: 9145
           name: aerospike-prometheus-exporter

 .
 .
 .
```

> Create or update your clusters after the Prometheus exporter sidecar is added.

Reviewer: ```suggestion
Create or update your clusters after you add the Prometheus exporter sidecar.
```

## Prometheus Configuration

Configure Prometheus to add exporter endpoints as scrape targets.

> If Prometheus is also running on Kubernetes, it can be configured to extract exporter targets from the Kubernetes API.

Reviewer: ```suggestion
If Prometheus is also running on Kubernetes, you can configure it to extract exporter targets from the Kubernetes API.
```

> In the following example, Prometheus will be able to discover and add exporter targets in the `default` namespace which has endpoint port name of `aerospike-prometheus-exporter`.

Reviewer: ```suggestion
In the following example, Prometheus discovers and adds exporter targets in the `default` namespace which has endpoint port name of `aerospike-prometheus-exporter`.
```

      action: keep
```

> See the Aerospike Database documentation for more information on installing and configuring the Aerospike Monitoring Stack.

Reviewer: ```suggestion
See [Aerospike Monitoring Stack documentation] (https://docs.aerospike.com/docs/tools/monitorstack/index.html)  for more information on installing and configuring the Aerospike Monitoring Stack.
```
